### ISSUE TITLE
样式不能在小程序上正常解析

### ISSUE THREAD

**Issue by seechangan** [surmon-china/vue-quill-editor#339]
使用vue-quill-editore富文本编辑器编辑出来的样式无法正常的在小程序中被解析，查看了一下标签上的  原来是以class类名形式来实现的，不是以行内样式编写的，请问如何解决 在小程序上无法正常解析css样式的问题。

**Comment by seechangan** [comment]
"vue": "^2.5.17"
"vue-ueditor-wrap": "^2.4.0"

**Comment by qinglingzhiyu** [comment]
我也遇到这个问题 该怎么解决

**Comment by asura0129** [comment]
同样遇到这个问题！

**Comment by seechangan** [comment]
只能换另外一个插件了，这个插件是没办法解决这个问题的。




------------------ 原始邮件 ------------------
发件人: "asura0129"<notifications@github.com>; 
发送时间: 2019年6月25日(星期二) 中午11:11
收件人: "surmon-china/vue-quill-editor"<vue-quill-editor@noreply.github.com>; 
抄送: "seechangan"<1071175587@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [surmon-china/vue-quill-editor] 样式不能在小程序上正常解析 (#339)




同样遇到这个问题！

**Comment by seechangan** [comment]
只能换一个富文本插件了，不然没法解决。




------------------ 原始邮件 ------------------
发件人: "qinglingzhiyu"<notifications@github.com>; 
发送时间: 2019年6月25日(星期二) 中午11:11
收件人: "surmon-china/vue-quill-editor"<vue-quill-editor@noreply.github.com>; 
抄送: "seechangan"<1071175587@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [surmon-china/vue-quill-editor] 样式不能在小程序上正常解析 (#339)




我也遇到这个问题 该怎么解决

**Comment by LiXinyu0926** [comment]
同样遇到这样的问题，项目里编辑器使用很频繁，不想换编辑器啊

**Comment by LiXinyu0926** [comment]
同遇到一样的问题

**Comment by seechangan** [comment]
好像只能换了。。。  那个编辑器是没有提供配置选项的 他的css样式都是以class定义的  或许你可以尝试着把它的对应css文件拷贝一份在小程序全局引入， 




------------------ 原始邮件 ------------------
发件人: "李心雨"<notifications@github.com>; 
发送时间: 2019年7月15日(星期一) 上午10:30
收件人: "surmon-china/vue-quill-editor"<vue-quill-editor@noreply.github.com>; 
抄送: "seechangan"<1071175587@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [surmon-china/vue-quill-editor] 样式不能在小程序上正常解析 (#339)




同样遇到这样的问题，项目里编辑器使用很频繁，不想换编辑器啊

**Comment by seechangan** [comment]
> 我也遇到这个问题该怎么解决

好像只能换了...那个编辑器是没有提供配置选项的他的css样式都是以类定义的或许你可以尝试着它的对应css文件拷贝一份在小程序全局引入  注意一下他的最外成的类名， 我看它很多样式都是基于那个类名之后再编写css 比如说 #editor .edit{ color:red; }

**Comment by seechangan** [comment]
### 

> 

> 

> 同遇到一样的问题
好像只能换了...那个编辑器是没有提供配置选项的他的css样式都是以类定义的或许你可以尝试着它的对应css文件拷贝一份在小程序全局引入注意一下他的最外成的类名，我看它很多样式都是基于那个类名之后再编写css比如说#editor .edit {color：red; }

**Comment by LiXinyu0926** [comment]
> > 
> 
> > 
> 
> > 遇到同一样的问题
> > 好像只能换速了...那个编辑器是没有提供配置选项的他的CSS样式都是以类定义的或许你可以尝试着它的对应的css文件拷贝一份在小程序全局引入注意一下他的最外成的类名，我看它很多样式都是基于那个类名之后再编写css比如说#editor .edit {color：red; }

对 我用的是wxparse来转换编辑器的内容变成小程序可用的，它转换过来不可展现的样式，我就根据quill的类名在wxparse的wxss的文件里添加样式了，这也是没办法的办法，但不想换编辑器的话这个办法确实可行。

**Comment by liujihaozhy** [comment]
坑

**Comment by xiaopz** [comment]
还没解决吗？
